OrbitGl/GlCanvas.cpp

```diff
@@ -138,6 +138,13 @@ void GlCanvas::EnableImGui() {
   }
 }
 
+orbit_accessibility::AccessibleInterface* GlCanvas::Accessibility() {
```

Comment:
We tend to use the GetOrCreateXYZ pattern for lazily initialized objects, consider using the same pattern here, GetOrCreateAccessibilityInterface?

Reply:
Should we use "*Accessible*Interface" rather than "*Accessibility*Interface"? It seems to me that the latter would be more appropriate.

Reply:
Good point about GetOrCreate, I'll change that. Regarding "Accessibility" vs "Accessible": I've named the interfaces "Accessible" to have them named similar to the Qt convention, and following the idea of "this is the interface for an accessible object". Also because "Accessible" is slightly less annoying to type than "Accessibility". I've named the module "OrbitAccessibility" though because it contains all the general concepts for Accessibility.

That being said - the method should probably be called `GetOrCreateAccessibleInterface`.

Reply:
Sounds good!

Reply:
I think it depends - in case of the tracks, the argument was rather that Tracks shouldn't just be created "on the fly", there usually should be one entity in charge of creating a track when it's needed. 

In this case here, the `accessibility_` could already be created in the constructor, it usually should just always exist. But since I'm using a virtual factory method that may be overridden in the inheriting class, I can't call it in the base constructor.  I could have added an `Init()` method that is called after construction, but I'm not a fan of additional initialization-calls because they violate RAII. In addition, there already is an `Initialize()` method in GlCanvas, but this may be called later because it's called on OpenGl initialization... so I decided that constructing the instance on demand is the cleanest way here.